### ISSUE TITLE
Track projects/submission status changes in ingest backoffice

### ISSUE THREAD

**Issue by gabsie** [ebi-ait/dcp-ingest-central#547]
As a data wrangler, 
I want to see when a Project changed statuses, 
so we can later report on these in terms of what happened and time it took. 

See [wireframes](https://miro.com/app/board/uXjVOWr0iR0=/)

related to #604 that records the stakeholder's point of view

Acceptance criteria:
---------------------
Given a data wrangler/stakeholder visits the project
When they open up the History tab
They can view the project lifecycle events (What, by Who, When)

As a start, we will focus on: The project changed statuses  (wrangler driven)
The submissions per project changed statuses (system driven)

**Comment by ofanobilbao** [comment]
I need to think about this from the stakeholder perspective. But I believe that as the number of projects we push to DCP grows, it will be critical that we store a log or record (visible to wranglers and stakeholders) of what changes have been applied to a Project, and whether the changes have been exported to DCP or archives or not.

**Comment by ofanobilbao** [comment]
related to #604

**Comment by yusra-haider** [comment]
as per @ofanobilbao  and @amnonkhen discussion, we will be capturing and displaying lifecycle events, i.e. state changes for a submission in this ticket

Posting the tentative UI design here for feedback, @gabsie 

![image.png](https://images.zenhubusercontent.com/5fa15836bb6093b67034928a/f4e80677-9b12-4b43-9a93-344e02fde0e5)

**Comment by amnonkhen** [comment]
@yusra-haider please format it as a table

**Comment by MightyAx** [comment]
@gabsie and @yusra-haider to talk about this today.

**Comment by gabsie** [comment]
Needs a design discussion meeting to minimise misunderstanding and define MVP

**Comment by MightyAx** [comment]
Current statuses in Ingest Back Office:
![Screenshot 2022-04-20 at 11.25.45.png](https://images.zenhubusercontent.com/5d4053c054f6346037c2ef53/184bf9cc-a22b-4d93-950c-2206ef4eb3ba)
![Screenshot 2022-04-20 at 11.25.57.png](https://images.zenhubusercontent.com/5d4053c054f6346037c2ef53/29736e52-5db1-4ba0-a2c2-d4425c60bd2a)

**Comment by prabh-t** [comment]
Not much progress on this on Friday, progress expected today.

**Comment by ofanobilbao** [comment]
Yusra is working on it

**Comment by prabh-t** [comment]
Yusra has draft PR ready for this. Requires some cleanup and ready for review.

**Comment by MightyAx** [comment]
@amnonkhen and @MightyAx to review draft PR's today.

**Comment by yusra-haider** [comment]
deployed in dev and staging now, and ready for wrangler testing

**Comment by MightyAx** [comment]
Core on production

**Comment by amnonkhen** [comment]
@yusra-haider , here are some comments after trying out this feature:
1. looks very nice
2. tab numbering problem, 2 "5"s
![image.png](https://images.zenhubusercontent.com/600ed297091d3ea152b83c2d/58ba4f5c-03d1-40f2-8135-6648bc6343da)

3. history table look and feel is inconsistent with metadata and data tables

**Comment by yusra-haider** [comment]
@amnonkhen 
for 2. fixed the tab numbering issue in: https://github.com/ebi-ait/ingest-ui/pull/174, and merged into dev. Waiting on gitlab pipelines

for 3. The table follows the same format as the contributors table, do we want it to follow a different format?
cc: @gabsie

**Comment by yusra-haider** [comment]
merged to master
